### PR TITLE
Edit profile

### PR DESCRIPTION
**Pull request by ishancoder**
Hi, Guys i've created Edit Profile Page please review it and let me know if any changes are to be made.

### CODE REVIEW

**Comment by razat249** [comment]
@ishancoder Also add the UI screenshot of your page for UI review. :smile:

File: .eslintrc.json [hunk 1]
```diff
@@ -31,6 +31,10 @@
         "semi": [
             "error",
             "always"
+        ],
```
Comment (razat249) [inline .eslintrc.json:34]: great :+1:

File: src/components/common/Pill/Pill.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./PillStyles.css";
+
+class Pill extends React.Component {
+    render() {
+        return(
+            <div className="pill-component">{this.props.children} <span className={this.props.cross === "true" ? "glyphicon glyphicon-remove" : ""}></span></div>
```
Comment (razat249) [inline src/components/common/Pill/Pill.jsx]: Split in multiple lines

File: src/components/common/DropDown/DropDown.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./DropDownStyles.css";
+
+class DropDown extends React.Component {
+    render() {
+        let dropDownElems = [];
+
+        for (let i = 0; i < this.props.contents.length; i++) {
```
Comment (razat249) [inline src/components/common/DropDown/DropDown.jsx]: Research if this is the right way to use for loop or any other functionality code(I mean inside the render function). There are other ways like constructor and react Component life cycle etc. Learn about them and also learn when to use what.

File: src/components/common/SkillArea/SkillArea.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Pill from "../Pill/Pill"; //eslint-disable-line no-unused-vars
+import "./SkillAreaStyles.css";
+
+class SkillArea extends React.Component {
+    render() {
+        let skillArray = this.props.skillSet;
```
Comment (razat249) [inline src/components/common/SkillArea/SkillArea.jsx:7]: Also confirm it here if this is right and in every component do the same.

**Review by razat249 — CHANGES_REQUESTED** [review]
Everything seems nice and working perfectly. Just need some research and changes. nice work :+1:

File: .eslintrc.json (hunk 1 above)
Comment (ishancoder) [reply .eslintrc.json:34]: Thanks

File: src/components/common/Pill/Pill.jsx (hunk 2 above)
Comment (ishancoder) [reply src/components/common/Pill/Pill.jsx]: Done!

File: src/components/common/DropDown/DropDown.jsx (hunk 3 above)
Comment (ishancoder) [reply src/components/common/DropDown/DropDown.jsx]: as thinkster.io this the current best practice however I've changed this for loop to JavaScripts Map function.

File: src/components/common/SkillArea/SkillArea.jsx (hunk 4 above)
Comment (ishancoder) [reply src/components/common/SkillArea/SkillArea.jsx:7]: Same here!

**Comment by ishancoder** [comment]
Hi, Reviewer I've made appropriate changes as requested. Thanks for reviewing.

**Review by razat249 — APPROVED** [review]
Nicely done

**Pull request merged**